packages/logo/src/Logo.php

```diff
@@ -44,4 +44,32 @@ public function render() {
 			esc_attr( 'Jetpack.' )
 		);
 	}
+
+	/**
+	 * Return string containing the Jetpack logo.
+	 *
+	 * @since 7.5.0
```

Comment:
Having a `@since` with the Jetpack version here is confusing - is this the package version, or the Jetpack version? We could remove it, but it would also make sense to keep the package version, or even better, specify what version specifically we're talking about here (Jetpack or package version).

Reply:
Good point. I suppose we're going to need to start thinking in terms of package versions instead of Jetpack versions, since Jetpack is going to be a collection of packages. For that reason, I think it should be the package version.

Reply:
@gititon let's write a quick PR or open issues for things like these, so we don't get lost in the chaos